[draft1]correct the attn naming for `UNet2DConditionModel`

yiyixuxu opened the pull request:
In this PR, we:
1. deprecate `attention_head_dim` and introduced a new variable `attention_head_dimension`
2.  the user to use `num_attention_heads`or `attention_head_dimension` in the configuration file for UNet2DConditionModel
3. make sure all the blocks accept both `num_attention_heads` and `attention_head_dim` arguments
4. when the deprecated `attention_head_dim` argument is passed to UNet2DConditionModel: we use `correct_incorrect_names()` function to calculate `num_attention_heads` and `attention_head_dimension`
5. in UNet2DConditionModel, after taking the above steps, calculate `attention_head_dim` based on `num_attention_head`, or calculate `num_attention_head` based on `attention_head_dimension` 
6. pass both arguments to all the blocks

I updated `ControlNetModel` with the same logic and also removed the `#Copied from` from the deprecated modeling_text_unet.py 

related to this PR https://github.com/huggingface/diffusers/pull/6983, which I like more because code is easier to understand



### quick sanity test
```python
import torch
from diffusers import UNet2DConditionModel
from diffusers.models.attention_processor import Attention

print(" ")
print(" SD1.5 (current mode configure file)")
repo = "runwayml/stable-diffusion-v1-5"
unet = UNet2DConditionModel.from_pretrained(repo, subfolder="unet", torch_dtype=torch.float16, variant="fp16")
for name, module in unet.named_modules():
    if isinstance(module, Attention):
        print(f"  module.inner_dim/module_heads:{module.inner_dim/module.heads}, module.heads:{module.heads}")
        print(f" module.scale: {module.scale}")

print(" ")
print(" SD1.5 (correct configure)")
repo = "runwayml/stable-diffusion-v1-5"
unet = UNet2DConditionModel.from_pretrained(repo, subfolder="unet", torch_dtype=torch.float16, variant="fp16", attention_head_dim=None, num_attention_heads=8, attention_head_dimension=None)
for name, module in unet.named_modules():
    if isinstance(module, Attention):
        print(f"  module.inner_dim/module_heads:{module.inner_dim/module.heads}, module.heads:{module.heads}")
        print(f" module.scale: {module.scale}")

print(f" ")
print(f" IF (current model configuration file)")
repo = "DeepFloyd/IF-I-XL-v1.0"
unet = UNet2DConditionModel.from_pretrained(repo, subfolder="unet", torch_dtype=torch.float16, variant="fp16")
for name, module in unet.named_modules():
    if isinstance(module, Attention):
        print(f"  module.inner_dim/module_heads:{module.inner_dim/module.heads}, module.heads:{module.heads}")
        print(f" module.scale: {module.scale}")

print(f" ")
print(f" IF (correct configure)")
repo = "DeepFloyd/IF-I-XL-v1.0"
unet = UNet2DConditionModel.from_pretrained(repo, subfolder="unet", torch_dtype=torch.float16, variant="fp16", attention_head_dim=None, num_attention_heads=None, attention_head_dimension=64)
for name, module in unet.named_modules():
    if isinstance(module, Attention):
        print(f"  module.inner_dim/module_heads:{module.inner_dim/module.heads}, module.heads:{module.heads}")
        print(f" module.scale: {module.scale}")
```

```
 SD1.5 (current mode configure file)
/home/yiyi_huggingface_co/diffusers/src/diffusers/models/unets/unet_2d_condition.py:249: FutureWarning:  `attention_head_dim` is deprecated and will be removed in a future version. Use `num_attention_heads` and `attention_head_dimension` instead.
  deprecate("attention_head_dim not None", "1.0.0", deprecation_message, standard_warn=False)
corrected potentially incorrect arguments, the model will be configured with `num_attention_heads` 8 and `attention_head_dimension` None.
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
 
 SD1.5 (correct configure)
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
 
 IF (current model configuration file)
corrected potentially incorrect arguments, the model will be configured with `num_attention_heads` None and `attention_head_dimension` 64.
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
 
 IF (correct configure)
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
```

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/diffusers/pr_6980). All of your documentation changes will be reflected on that endpoint. The docs are available until 30 days after the last update.

yiyixuxu commented:
testing popular checkpoints 
```python
from diffusers import UNet2DConditionModel
import torch
from diffusers.models.attention_processor import Attention

repos = [
    "runwayml/stable-diffusion-v1-5",
    "stabilityai/stable-diffusion-xl-base-1.0",
    "DeepFloyd/IF-I-XL-v1.0",
    "kandinsky-community/kandinsky-2-2-decoder",
    "kandinsky-community/kandinsky-2-1",
    "stabilityai/sd-x2-latent-upscaler",
    "segmind/SSD-1B"
]

for repo in repos:
    print(f" ")
    print(f" {repo} (current model configuration file)")
    unet = UNet2DConditionModel.from_pretrained(repo, subfolder="unet", torch_dtype=torch.float16)
    for name, module in unet.named_modules():
        if isinstance(module, Attention):
            print(f"  module.inner_dim/module_heads:{module.inner_dim/module.heads}, module.heads:{module.heads}")
            print(f" module.scale: {module.scale}")
```

output
```
 runwayml/stable-diffusion-v1-5 (current model configuration file)
/home/yiyi_huggingface_co/diffusers/src/diffusers/models/unets/unet_2d_condition.py:249: FutureWarning:  `attention_head_dim` is deprecated and will be removed in a future version. Use `num_attention_heads` and `attention_head_dimension` instead.
  deprecate("attention_head_dim not None", "1.0.0", deprecation_message, standard_warn=False)
corrected potentially incorrect arguments, the model will be configured with `num_attention_heads` 8 and `attention_head_dimension` None.
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:80.0, module.heads:8
 module.scale: 0.11180339887498948
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:40.0, module.heads:8
 module.scale: 0.15811388300841897
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
  module.inner_dim/module_heads:160.0, module.heads:8
 module.scale: 0.07905694150420949
 
 stabilityai/stable-diffusion-xl-base-1.0 (current model configuration file)
corrected potentially incorrect arguments, the model will be configured with `num_attention_heads` [5, 10, 20] and `attention_head_dimension` None.
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
 
 DeepFloyd/IF-I-XL-v1.0 (current model configuration file)
corrected potentially incorrect arguments, the model will be configured with `num_attention_heads` None and `attention_head_dimension` 64.
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:33
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:22
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:44
 module.scale: 0.125
 
 kandinsky-community/kandinsky-2-2-decoder (current model configuration file)
corrected potentially incorrect arguments, the model will be configured with `num_attention_heads` None and `attention_head_dimension` 64.
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:18
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:18
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:18
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:18
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:18
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:18
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:18
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
 
 kandinsky-community/kandinsky-2-1 (current model configuration file)
corrected potentially incorrect arguments, the model will be configured with `num_attention_heads` None and `attention_head_dimension` 64.
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:18
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:18
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:18
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:18
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:18
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:18
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:18
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:24
 module.scale: 0.125
 
 stabilityai/sd-x2-latent-upscaler (current model configuration file)
corrected potentially incorrect arguments, the model will be configured with `num_attention_heads` None and `attention_head_dimension` 64.
`num_attention_heads` argument is passed but ignored. The number of attention heads is determined by `attention_head_dim`, `in_channels` and `out_channels`.
`num_attention_heads` argument is passed but ignored. The number of attention heads is determined by `attention_head_dim`, `in_channels` and `out_channels`.
`num_attention_heads` argument is passed but ignored. The number of attention heads is determined by `attention_head_dim`, `in_channels` and `out_channels`.
  module.inner_dim/module_heads:64.0, module.heads:6
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:6
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:6
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:6
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:12
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:6
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:6
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:6
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:6
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:6
 module.scale: 0.125
 
 segmind/SSD-1B (current model configuration file)
corrected potentially incorrect arguments, the model will be configured with `num_attention_heads` [5, 10, 20] and `attention_head_dimension` None.
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:20
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
  module.inner_dim/module_heads:64.0, module.heads:10
 module.scale: 0.125
```
